domain/src/main/java/com/knocklock/domain/usecase/setting/GetUserTypeUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.knocklock.domain.usecase.setting
+
+import com.knocklock.domain.repository.UserRepository
+import javax.inject.Inject
+
+class GetUserTypeUseCase @Inject constructor(
+    private val userRepository: UserRepository
+) {
+    operator fun invoke() = userRepository.getUser()
+}
```

Comment:
Type은 빼도 될것 같네요?? 👁️ 
```suggestion
class GetUserUseCase @Inject constructor(
    private val userRepository: UserRepository
) {
    operator fun invoke() = userRepository.getUser()
}
```

Reply:
찬성입니다~